plugins/org.polarsys.capella.transition.system2subsystem/src/org/polarsys/capella/transition/system2subsystem/preferences/PropertyValuesPreference.java

```diff
@@ -159,7 +160,8 @@ public void setValue(IPropertyContext context_p) {
       }
 
       if (archi != null && archi.eContainer() instanceof SystemEngineering) {
-
+        if (archi.eContainer().eContainer() instanceof Project)
+          visit(values, archi.eContainer().eContainer());

```

Comment:
With a visit on the root Project, i think if you do a transition from a logical component, you will also have the property values from physical layer (preventing the code below to restrict only on upper layers "retrieve the lowest level architecture of all selected elements so that we return all valid PV's")
no? wdyt